hc/accounts/tests/test_profile.py

```diff
@@ -108,3 +116,27 @@ def test_it_shows_badges(self):
         self.assertNotContains(r, "bobs-tag.svg")
 
     ### Test it creates and revokes API key
+    def test_it_creates_api_key(self):
+        self.client.login(username="alice@example.org", password="password")
+
+        self.client.get("/accounts/profile/")
+
+        # Set the API key, in this case "abc"
+        self.alice.profile.refresh_from_db()
+        api_key = self.alice.profile.api_key
+
+        # Assert that the API key is not None
+        self.assertIsNotNone(api_key)
```

Comment:
add an assertion that the response contains the phrase `The API key has been created`. check out the file `hc/accounts/views.py` lines 146-153.
additionally you should create an api_key by making a POST request to `/accounts/profile/` with a payload of the form `form = {"create_api_key": "<some_string>"}`.
Currently the api key you are testing against was created during the setup in `hc/test.py` and is hence not a test of the method of creating an api key.